client/components/wizard/README.md

```diff
@@ -0,0 +1,62 @@
+Wizard
+======
+`Wizard` is a component that leads the user through a series of pre-defined steps.
+It keeps track of progress and enables navigating back or forward through the steps.
+
+## Usage
+
+```js
+const components = {
+	'first': <First />,
+	'second': <Second />,
+};
+const steps = [ 'first', 'second' ];
+
+<Wizard
+	basePath="/section/wizard"
+	components={ components }
+	steps={ steps }
```

Comment:
I'd be worried about using an object for that, but a [`Map`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Map) could work. Since maps aren't indexed though, there would need to be some looping involved to find the index of the current step and when navigating backward and forward between steps.